src/test/ui/parser/incorrect-move-async-order-issue-79694.stderr

```diff
@@ -0,0 +1,13 @@
+error: the order of `move` and `async` is incorrect
+  --> $DIR/incorrect-move-async-order-issue-79694.rs:7:13
+   |
+LL |     let _ = move async { };
+   |             ^^^^^^^^^^
+   |
+help: try switching the order
```

Comment:
We should show the correct one rather than try switching the order. I opened a pull request for the order of pub async on function, that shows did you mean xxx yyy, easier to follow compared to 'switching the order'.

Reply:
I requested this wording to remain consistent [with other diagnostics](https://github.com/rust-lang/rust/blob/0d97f7a96877a96015d70ece41ad08bb7af12377/src/test/ui/mut/mut-ref.stderr#L5). In the suggestion that follows the message, it does show the correct order. I don't mind seeing our wording here change, if you'd like to submit a PR for that, but I'd like to see all of the relevant diagnostics changed for consistency.

Reply:
> error: the order of `mut` and `ref` is incorrect

But what if there are more than 2 items? Say, `pub async const fn` becomes `async const pub fn`? So we say `the order of `pub` and `async` and `const` is incorrect?